docs/development/building/Building-in-Mac-OSX.md

```diff
@@ -68,6 +68,6 @@ changes from the repository, then rebuild the firmware:
 git reset --hard
 git pull
 
-make clean TARGET=MATEKH743
-make TARGET=MATEKH743
+make clean
```

Comment:
make CONFIG=MATEKH743

Reply:
- `rm -rf obj` is less  typing :)
- can we make this `make clean MATEKH743` and `make clean` for all?

Reply:
we have `make MATEKH743_clean` as well, so what id preferred? (or no preference?)

Reply:
`make MATEKH743_clean` is ugly :)